tests/cecil-tests/Helper.cs

```diff
@@ -105,6 +147,15 @@ public static string GetBCLDirectory (string assembly)
 			}
 		}
 
+		public static IEnumerable NetPlatformAssemblies {
+			get {
+				yield return new TestCaseData (Path.Combine (Configuration.GetDotNetRoot (), "Microsoft.iOS.Ref", "ref", "net6.0", "Xamarin.iOS.dll"));
+				yield return new TestCaseData (Path.Combine (Configuration.GetDotNetRoot (), "Microsoft.tvOS.Ref", "ref", "net6.0", "Xamarin.TVOS.dll"));
+				yield return new TestCaseData (Path.Combine (Configuration.GetDotNetRoot (), "Microsoft.macOS.Ref", "ref", "net6.0", "Xamarin.Mac.dll"));
+				yield return new TestCaseData (Path.Combine (Configuration.GetDotNetRoot (), "Microsoft.MacCatalyst.Ref", "ref", "net6.0", "Xamarin.MacCatalyst.dll"));
```

Comment:
I think you can simplify this now:

```suggestion
				return Configuration.GetRefLibraries ();
```

Reply:
done! Also added "Microsoft.MacCatalyst.Ref" to GetRefNuGetName which GetRefLibraries needs to work